questionpy_common/api/attempt.py

```diff
@@ -70,31 +74,40 @@ class ClassifiedResponse(BaseModel):
     score: float
 
 
+class ScoredInputState(Enum):
+    CORRECT = "CORRECT"
+    CONSEQUENTIAL_SCORE = "CONSEQUENTIAL_SCORE"
+    PARTIALLY_CORRECT = "PARTIALLY_CORRECT"
+    WRONG = "WRONG"
+
+
+class ScoredInputModel(BaseModel):
+    state: ScoredInputState
+    score: float | None = None
+    score_max: float | None = None
+    specific_feedback: str | None = None
+    right_answer: str | None = None
+
+
+class ScoredSubquestionModel(BaseModel):
+    score: float | None = None
+    score_final: float | None = None
+    scoring_code: ScoringCode | None = None
+    response_summary: str
+    response_class: str
+
+
 class ScoreModel(BaseModel):
-    scoring_state: str = "{}"
+    scoring_state: str | None = None
```

Comment:
Ein gescorter Attempt _musste_ bisher ein Scoring State haben. Das habe ich mal aufgeweicht, weil viele Pakete vermutlich eh `{}` verwenden würden und ich mir so später den Check `attempt.scoring_state is None` im Wrapper erspare. Ich glaube es spricht nichts dagegen, let me know falls ich etwas übersehen habe.